feat(env): add false option for envDir to disable env loading (#19503)
vitejs/vite: docs/config/shared-options.md

## envDir

- **Type:** `string | false`
- **Default:** `root`

> The directory from which `.env` files are loaded. Can be an absolute path, or a path relative to the project root.
> 
> See [here](/guide/env-and-mode#env-files) for more about environment files.
> 
> :::warning Warning
> If you don't want to use Vite's built-in `.env` file handling, you can set `envDir: false` to disable it.
> :::

Reviewer: ```suggestion
The directory from which `.env` files are loaded. Can be an absolute path, or a path relative to the project root. `false` will disable the `.env` file loading.

See [here](/guide/env-and-mode#env-files) for more about environment files.
```
I think it doesn't have to be a warning.
